docs/23_gitignore.md

```diff
@@ -0,0 +1,44 @@
+## Configure Git to ignore files
+
+You can add a text file in your repository titled `.gitignore` that instructs Git to stop tracking changes of certain files. This also means that these files will not be pushed to GitHub from your local machine.
+
+**Files that you might want Git to ignore:**
+1. Secrets, e.g. `*.pem`, `*.key`
+1. Binary files, e.g. `*.pdf*`, `*.docx`
+1. Log files, e.g. `*.log`
+1. Build files, e.g. `Build/*`
+
+### Here are some ways you can add a `.gitignore` file:
+
+#### Ignore files  locally
+
+In your Terminal or Git Bash, `cd` to the repository of interest and create a `.gitignore` file
+```
+touch .gitignore
```

Comment:
In general can we try to be more consistent when presenting the reader with steps to complete a task? In this specific instance, the user has created a `.gitignore` file, but hasn't committed it, added anything to the `.gitignore` file, etc. 

Ideally we would have some exercise that lets us show them how the `.gitignore` file works both from a "I can see how I could potentially use this" perspective and a "Oh, I can see it in action through this activity" perspective. 

Overall, great addition for this topic though 🎉

Reply:
This might be useful here: https://gist.github.com/octocat/9257657